cukinia

```diff
@@ -229,6 +230,17 @@ _cukinia_http_request()
 	wget -q -O /dev/null "$url"
 }
 
+# _cukinia_symlink: validate the target of symlink
+# args1: link
+# args2: target
+_cukinia_symlink()
+{
+	local link="$1"
+	local target="$2"
+	_cukinia_prepare "Checking for $link pointing to $target"
```

Comment:
I think Checking link "$link" points to "$target" makes more sense as a log message.